src/api/index.js

```diff
@@ -1,2 +1,3 @@
-require('../../babel-transformer');
+require('reify');
```

Comment:
This file is now useless, (now that you append --require reify...) so you can git mv server.js index.js